### ISSUE TITLE
Tree Tests

### ISSUE THREAD

**Issue by harshkpatel** [CSC207-2022F-UofT/course-project-the-best-monopoly-team#81]
A couple of Tree tests need to be fully written

**Comment by harshkpatel** [comment]
JavaDoc for some the tree elements also need to be written

**Comment by derrcodes** [comment]
JavaDoc is complete and majority of tests are finished. Still waiting on a few of them to be done.

Note: I think one of the files related to the unmortgage feature still doesn't have JavaDoc or tests but its because implementation is still not finished. Apart from that everything should be good to go.

**Comment by derrcodes** [comment]
Closing this issue because JavaDoc is complete. Making a new issue for the remaining tests mentioned above.
